src/main/java/com/db/carshop/employee/exceptions/EmployeeDoesNotExistException.java

```diff
@@ -0,0 +1,7 @@
+package com.db.carshop.employee.exceptions;
+
+public class EmployeeDoesNotExistException extends RuntimeException{
```

Comment:
não sei se é necessário ou a prática aqui... porem gerar exception forçadas mesmo que para tratar business , não é uma boa pratica. Talvez trocar para domain events seria legal (sugestão)

Reply:
O Gabriel tinha me ensinado assim, ,mas é uma boa sugestão pro futuro